.github/workflows/ci.yml

```diff
@@ -74,13 +74,20 @@ jobs:
         pyver: ['3.7', '3.8', '3.9', '3.10']
         no-extensions: ['', 'Y']
         os: [ubuntu, macos, windows]
+        experimental: [false]
         exclude:
           - os: macos
             no-extensions: 'Y'
           - os: windows
             no-extensions: 'Y'
+        include:
+          - os: ubuntu
+            pyver: "3.11.0-alpha - 3.11.0"
+            experimental: true
```

Comment:
FWIW, if it passes, maybe we don't need "experimental" anymore? The first release candidate is just a week away, after all...

Reply:
I'm inclined to just leave an experimental version in, so it can just be bumped to 3.12 a little later.

Reply:
Too late https://github.com/aio-libs/frozenlist/commit/15b00857f004dc44ab9610695026188868bddb0e